deployment/helm/storage-ui/config/org.opencadc.vosui.properties

```diff
@@ -29,9 +29,17 @@ org.opencadc.vosui.{{ $name }}.service.features.paging = {{ required "The servic
 org.opencadc.vosui.gms.service_id = {{ required "The storageUI.gmsID string is required." .Values.deployment.storageUI.gmsID }}
 org.opencadc.vosui.theme.name = {{ required "The storageUI.themeName string is required (valid values are src or canfar)." .Values.deployment.storageUI.themeName }}
 
+{{ if .Values.deployment.storageUI.oidc.existingSecret -}}
```

Comment:
Would you please change `existingSecret` to `existingSecretName`, just to underline the difference that this is a Kubernetes secret, and not an IAM client secret?

Reply:
Good idea, done 👍